src/main/drivers/adc_stm32f7xx.c

```diff
@@ -83,6 +104,113 @@ const adcTagMap_t adcTagMap[] = {
     { DEFIO_TAG_E__PA7, ADC_DEVICES_12,  ADC_CHANNEL_7  },
 };
 
+void adcInitDevice(adcDevice_t *adcdev, int channelCount)
+{
+    adcdev->ADCHandle.Init.ClockPrescaler        = ADC_CLOCK_SYNC_PCLK_DIV8;
```

Comment:
Could be beneficial to move to LL here as well

Reply:
Exactly. I've found there are so much extra code executed in ADC HAL. I'll stack the LL conversion to my todo list, but priority is rather low as injection is only called once a second (1Hz).